torch/csrc/autograd/autograd_not_implemented_fallback.cpp

```diff
@@ -528,24 +528,38 @@ static void autogradNotImplementedInplaceOrViewFallbackImpl(
   if (is_view) {
     c10::IValue& aliased_output_iv =
         (*stack)[stack->size() - num_returns + aliased_output_idx];
+
+    // See NOTE [ View + Inplace detection ] for more details about this logic
+    const auto erroring_view_func = [op_name = op_name](const at::Tensor&) {
+      // We always need this view_func because otherwise if we do in-place
+      // on this view, we would implicitly use AsStridedBackward instead
+      // of the NotImplemented node. For the cross-dtype/non-strided
+      // cases, we would create something like this anyway
+      TORCH_CHECK(
+          false,
+          "Mutating the view ",
+          op_name,
+          " which does not have a derivative implemented is forbidden.");
```

Comment:
I don't think we would actually want to error here for the rev_view case if we want this view to work for PT2.
Would be nice to update this error message though, since folks can now reach this path without doing in-place over view.

Reply:
You're right: for `rev_view_func()` we can reach this path without an in-place over a view. I expect it to (eventually) be hit when attempting to meta-ify a view output of an op that utilizes this fallback. So at the very least, there should be a separate error msg for `rev_view_func()`.

Maybe the error message should mention this scenario specifically? cc @albanD wdyt?

Reply:
For posterity: resolved with separate error msg for `rev_view_func()`